core/src/test/resources/com/squarespace/template/plugins/platform/f-variants-select-4.html

```diff
@@ -1,12 +1,12 @@
 :JSON
 {
   "localizedStrings": {
-    "giftCardVariantSelectText": "Get foobar"
+    "productVariantSelectText": "Get foobar"
```

Comment:
Is this change intended ?

Reply:
Yes, but it depends. The intention of this test originally appeared to be testing gift card behavior. See product type = 4 but after the changes I have made, the titles of the variants will not take the {name} it will instead take the translated value of "Value" so this test would change quite a bit if I were to modify it for a gift card. Instead, I switched it to a physical product test but can revert it back / make modifications to test it as a gift card.